WTWSMS/events/religious_events.txt

```diff
@@ -3872,42 +3923,47 @@ character_event = {
 			factor = 1
 			# Traits
 			modifier = {
-				factor = 0.3
+				factor = 0.2
 				trait = zealous
 			}
 			modifier = {
-				factor = 3
+				factor = 4
 				trait = cynical
 			}
 			modifier = {
-				factor = 2
+				factor = 4
 				trait = arbitrary
 			}
 			modifier = {
-				factor = 3
+				factor = 4
 				FROM = { religion_group = christian }
 				trait = sympathy_christendom
 			}
 			modifier = {
-				factor = 3
+				factor = 4
 				FROM = { religion_group = muslim }
 				trait = sympathy_islam
 			}
 			modifier = {
-				factor = 3
+				factor = 4
 				FROM = { religion_group = jewish_group }
 				trait = sympathy_judaism
 			}
 			modifier = {
-				factor = 3
+				factor = 4
 				FROM = { religion_group = zoroastrian_group }
 				trait = sympathy_zoroastrianism
 			}
 			modifier = {
-				factor = 3
+				factor = 4
```

Comment:
Compare with the three elsewhere and you will see it is already significant. Maybe too much even.